src/main/resources/lib/office-league-slack.js

```diff
@@ -15,15 +15,18 @@ function createFinishedGameMessage(data) {
     };
 
     let matchResult = "";
```

Comment:
Why did you initialize it like that? it's value will always be replaced in one of the two cases in the conditional at line 20, so isn't it better to initialize it like `let matchResult;`?

Reply:
I think i did matchResult += in a previus version.